### PR TITLE
reordering of Dockerfile and adding comments

### PR DESCRIPTION
**Pull request by twnolan**
- [x] Reordering the Dockerfile to avoid re-downloading the keycloak server again if a change is made to the raincatcher-realm.json file
- [x] Added additional comments to the Dockerfile 
- [x] Updated keycloak password

### CODE REVIEW

**Comment by twnolan** [comment]
@TommyJ1994 - can you review this please.

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ LABEL maintainer="thnolan@redhat.com"
 LABEL name="raincatcher-keycloak-docker"
 LABEL version="1.0.0"
 
+# Enviornment Variables
 ENV KEYCLOAK_VERSION 3.1.0.Final
 ENV KEYCLOAK_ADMIN_USERNAME admin
 ENV KEYCLOAK_ADMIN_PASSWORD Password1
```
Comment (tomjackman) [inline Dockerfile]: @twnolan Can we update the admin's password to `admin` too? Its the one I typically use and think its the one used in the keycloak docs.

Comment (twnolan) [reply Dockerfile]: @TommyJ1994 done

**Comment by twnolan** [comment]
@TommyJ1994 Password is updated now

**Review by tomjackman — APPROVED** [review]
LGTM :+1:

**Pull request merged**